vtex/toolbelt
# Use @vtex/node-error-report

#### What is the purpose of this pull request?
Use @vtex/node-error-report npm package to centralize error parsing, serialization and sanitization on toolbelt and node-vtex-api. 

#### How should this be manually tested?
```
git checkout chore/improve-errors

yarn watch
```

Go to a master workspace and try `vtex-test workspace reset`

An errorID has to be printed - this can be searched on splunk:
```
index=io_vtex_logs $errorID
```
Check the fields validity logged on splunk:
- Request info
- Your environment
- The error metadata

#### Types of changes

#### Chores checklist
